app/src/main/java/at/roteskreuz/stopcorona/model/repositories/NotificationsRepository.kt

```diff
@@ -74,12 +77,25 @@ interface NotificationsRepository {
      * the Exposure Notifications Framework was too low to qualify for a quarantine.
      */
     fun displayNotificationForLowRisc()
+
+    /**
+     * Display a notification, reminding the user to upload the keys from the day before.
+     * Context: For privacy reasons, the key from today is not accessible until tomorrow.
+     */
+    fun displayNotificationForUploadingKeysFromTheDayBefore()
+
+    /**
+     * observe
+     */
+    fun checkAndScheduleMissingKeysNotification()
```

Comment:
I would suggest to move this method to `ExposureNotificationManager`.
Btw, update KDoc please.

Reply:
Resolved together.